test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestOpenJ9DiagnosticsMXBean.java

```diff
@@ -263,10 +264,30 @@ private void testLocal_setDumpOptions() {
 			e.printStackTrace();
```

Comment:
The same here - let this code throw and TestNG can handle the exception

Reply:
Thanks, I have modified the test to throw the exception.